task2_contains_duplicate.py

```diff
@@ -0,0 +1,13 @@
+
+
+def containsDuplicate2(nums):
+    """
+    Check duplicates in an array.
+    Return True if array consist duplicated values.
+    :type nums: List[int]
+    :rtype: bool
+    """
+    unique_num_length = len(set(nums))
+    num_length = len(nums)
+    return num_length > unique_num_length
```

Comment:
if we go with built-in structures to solve this task, then it makes sense to make a one-liner.
```suggestion
    return len(nums) > len(set(nums))
```
your version is good, as it is self-explanatory)

Reply:
btw, what is the complexity of this approach?

Reply:
when I write it as separate variables it works faster :)
complexity - O(n) because of set()